presto-main/src/main/java/com/facebook/presto/sql/analyzer/AggregationAnalyzer.java

```diff
@@ -408,6 +423,10 @@ protected Boolean visitDereferenceExpression(DereferenceExpression node, Void co
 
         private Boolean isField(QualifiedName qualifiedName)
         {
+            if (lambdaVariableNames.contains(qualifiedName.toString())) {
```

Comment:
@haozhun just a note: `columnReferences` _is_ an identity set most of the time and whenever it's not (e.g. in `AggregationAnalyzer` ATM on `master`), it's a bug. This is being fixed by #7354.